pkgs/desktops/gnome/misc/gnome-extensions-cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
+  pname = "gnome-extensions-cli";
+  version = "0.9.5";
+  format = "pyproject";
+
+  src = fetchPypi {
+    pname = "gnome_extensions_cli";
+    inherit version;
+    hash = "sha256-4eRVmG5lqK8ql9WpvXsf18znOt7kDSnpQnLfy73doy4=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    colorama
+    more-itertools
+    packaging
+    pydantic
+    requests
+  ] ++ typer.optional-dependencies.all;
+
+  # No tests available
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "gnome_extensions_cli"
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/essembeh/gnome-extensions-cli";
+    description = "Command line tool to manage your Gnome Shell extensions";
```

Comment:
GNOME should be capitalized.

Reply:
Consider it done. I'll edit it with the buildPythonApplication change.